cmd/operator-sdk/alpha/scorecard2/cmd.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package scorecard2
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+var (
+	dsl      string
+	bundle   string
+	selector string
+)
+
+//scorecard2 alpha command stub
```

Comment:
is this the correct comment for `NewCmd`? not sure, if linter cares.

Reply:
To follow godoc conventions it should be something like:
```
// NewCmd constructs and returns a new cobra command that runs scorecard
```

GoDoc comments always start with the name of the function, and they're expected on all exported functions. See https://blog.golang.org/godoc